src/main.js

```diff
@@ -120,21 +120,18 @@ if ('serviceWorker' in window.navigator) {
     });
 }
 
-window.setTimeout(function () {
-    
-    "use strict";
 
-    // XXXBramble: We host the filesystem in the hosting, parent window.  Make sure
-    // we're loaded within an iframe, and warn if not.
-    if (window.location === window.parent.location) {
+// XXXBramble: We host the filesystem in the hosting, parent window.  Make sure
+// we're loaded within an iframe, and warn if not.
+if (window.location === window.parent.location) {
+    $('#iframe-warning').css({
+      'visibility': 'visible'
+    });
+    $('.sk-spinner-rotating-plane.sk-spinner').css({
+      'visibility' : 'hidden'
+    });
 
-        window.document.write(
-            "<h1>Bramble must be hosted</h1>" +
-            "<p>Oops! Bramble was loaded directly, outside of an iframe. For security it must be hosted in an iframe.</p>" +
-            "<p>For local development, you can run Bramble in an iframe by using <a href=\"hosted.html\">hosted.html</a> instead of <a href=\"index.html\">index.html</a>.</p>"
-        );
-    }
-}, 1000);
+}
```

Comment:
Probably we should throw this remaining code in an `else` block, since it's not worth doing if the first `if` happens.